CONTRIBUTING.md

```diff
@@ -35,9 +36,32 @@ times in N different commits. If there was some accidental reformatting or white
 changes during the course of your commits, please rebase them away before submitting
 the PR.
 
+### CHANGELOG using Beachball
+
+[Beachball](https://microsoft.github.io/beachball/) is a semantic version bumper that also has an automated tool to ask devs to log changes in a simple CLI manner.
+
+The TeamsJS Client SDK v2 repo contains a changelog (file is named `CHANGELOG.md`) for substantial changes in the `<root>/packages/teams-js` directory. If you make any changes to the `<root>/packages/teams-js` directory (you can see the exception files and directories in `<root>/beachball.config.js`), you must run `yarn generate-change-files` or `yarn beachball` from the monorepo root to generate change files.
+
+Beachball generates JSON change files based on a few simple answers from you:
+
+- Change type: this can be one of four things. (we won't be allowing the Prerelease option in this repo at this time.) Patch, Minor, None, and Major.
```

Comment:
nit: Can you capitalize the "w" in "We won't be allowing..."?

Reply:
Actually, can you rephrase it as "The Prerelease option is not allowed in this repo at this time."

Reply:
Done, will be included in the next commit(s)!

Reply:
Can we remove mentioning the prerelease option altogether? We won't be using it now or any time later.

Reply:
And it won't really show up for people since we have it disabled in the beachball config

Reply:
Yes for sure. Upcoming in next commit.